OutOfSchool/OutOfSchool.WebApi/Extensions/MappingExtensions.cs

```diff
@@ -22,9 +22,9 @@ public static ApplicationDto ToModel(this Application application)
             {
                 cfg.CreateMap<Address, AddressDto>();
                 cfg.CreateMap<Teacher, TeacherDTO>();
-                cfg.CreateMap<Workshop, WorkshopDTO>()
-                .ForMember(dest => dest.Direction, opt => opt.MapFrom(src => src.Direction.Title))
-                .ForMember(dest => dest.Keywords, opt => opt.MapFrom(src => src.Keywords.Split('¤', StringSplitOptions.None)));
+                cfg.CreateMap<Workshop, WorkshopCard>()
+                    .ForMember(dest => dest.Direction, opt => opt.MapFrom(src => src.Direction.Title));
+                // .ForMember(dest => dest.Keywords, opt => opt.MapFrom(src => src.Keywords.Split('¤', StringSplitOptions.None)));
```

Comment:
unless it's something left for refactoring later (with a TODO) - application code should be without commented lines. If you need to remove code - delete it.